src/chess/Command.java

```diff
@@ -18,17 +18,14 @@
   static Command parse(String str) {
     switch (str.toLowerCase()) {
       case "help":
-        System.out.println("* type 'help' for help");
-        System.out.println("* type 'board' to see the board again");
-        System.out.println("* type 'resign' to resign");
-        System.out.println("* type 'moves' to list all possible moves");
-        System.out.println("* type a square (e.g. b1, e2) to list possible moves for that square");
-        System.out.println("* type a UCI (e.g. b1c3, e7e8q) to make a move");
         return HELP;
+
```

Comment:
Not a good idea to break code format
If every other part have a blank line, you can add it, but if not, you shouldn't